MediaBrowser.Model/Session/GeneralCommandType.cs

```diff
@@ -43,6 +43,11 @@ public enum GeneralCommandType
         Guide = 32,
         ToggleStats = 33,
         PlayMediaSource = 34,
-        PlayTrailers = 35
+        PlayTrailers = 35,
+        SetShuffleQueue = 36,
+        Playstate = 37,
```

Comment:
Playstate or PlayState?

Reply:
the latter according to @nielsvanvelzen I think

Reply:
See https://github.com/jellyfin/jellyfin/pull/4210 for that change